### DISCUSSION TITLE
串流的时候如何用手柄按出快捷菜单

### DISCUSSION THREAD

**Discussion by coro531** [ClassicOldSong/moonlight-android#34] [General]
阿西西版本moonlight可以手柄长按start调出快捷菜单，想问下黑月版能不能用手柄调出快捷菜单，因为是在电视上串流，没办法左滑返回调出快捷菜单，希望可以用手柄按出

目前长按start键是开启鼠标模式，是否能够有个选项可以选择长按start调出快捷菜单

![QQ截图20241011104957](https://github.com/user-attachments/assets/5959abde-107c-435e-99ee-ee677181322e)

**Comment by ClassicOldSong** [comment]
短按一下start然后长按再松开即可
